### DISCUSSION TITLE
我在使用yarn安装code-server,最终出现报错,如何解决呢?

### DISCUSSION THREAD

**Discussion by yuboabc** [coder/code-server#5612] [Q&A]
error /home/.config/yarn/global/node_modules/code-server: Command failed.
Exit code: 1
Command: sh ./postinstall.sh
Arguments: 
Directory: /home/.config/yarn/global/node_modules/code-server
Output:
Installing Code dependencies...
yarn.lock file not present, not running in development mode. use npm to install code-server!

在termux中安装的

**Comment by lg-nineoneninethreenine** [comment]
你运行了`yarn install`吗？

Written in Simplified Chinese(简体中文）

What I was telling was: Did you run yarn install?

**Comment by yuboabc** [comment]
运行了~~~
运行环境 termux 
node  16.14.0
python 3.10.7
依赖 gpft-20220711


| |
陈龙
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2022年10月2日 16:41 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [coder/code-server] 我在使用yarn安装code-server,最终出现报错,如何解决呢? (Discussion #5612) |

你运行了yarn install吗？

Written in Simplified Chinese(简体中文）

What I was telling was: Did you run yarn install?

**Reply by lg-nineoneninethreenine** [reply]
Hmm, I got this translation(helps to those who need it):
<details>

Running~~~ Operating environment termux node 16.14.0 python 3.10.7 Depends on gpft-20220711 | | Chen Long | | ***@***.*** | ---- Reply to the original email----  | From | ***@***.***> | | Sent Date | October 2, 2022 16:41 | | To | ***@***.***> |  | CC | ***@***.***> , ***@***.***> | | Subject | Re: [coder/code-server] I am using yarn to install code-  server, an error occurs in the end, how to solve it? (Discussion #5612) | Did you run yarn install?  Written in Simplified Chinese What I was telling was: Did you run yarn install? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: **  *@***.***>

</details>

@yuboabc 请等待进一步的信息，这似乎是一个相对较新的错误...我将与维护人员交谈并告诉是否有任何修复。你还好吗？


I asked @yuboabc to wait since this looks like a relatively new bug. Can any maintainers look at the translation collapsible above and tell if this is a new bug or it's just a simple thing?

Edit: weird Google Translate changed the "is it fine with you?" phrase in the end of the Simplified Chinese talk... Don't get confused, please.

**Comment by yuboabc** [comment]
OK 
No problem, if it is a new problem. Please repair, thank you
| |
陈龙
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2022年10月2日 22:58 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [coder/code-server] 我在使用yarn安装code-server,最终出现报错,如何解决呢? (Discussion #5612) |

Hmm, I got this translation(helps to those who need it):

Running~~~ Operating environment termux node 16.14.0 python 3.10.7 Depends on gpft-20220711 | | Chen Long | | @.*** | ---- Reply to the original email---- | From | @.> | | Sent Date | October 2, 2022 16:41 | | To | @.> | | CC | @.> , @.> | | Subject | Re: [coder/code-server] I am using yarn to install code- server, an error occurs in the end, how to solve it? (Discussion #5612) | Did you run yarn install? Written in Simplified Chinese What I was telling was: Did you run yarn install? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ** @.*>

@yuboabc 请等待进一步的信息，这似乎是一个相对较新的错误...我将与维护人员交谈并告诉是否有任何修复。你还好吗？

I asked @yuboabc to wait since this looks like a relatively new bug. Can any maintainers look at the translation collapsible above and tell if this is a new bug or it's just a simple thing?

**Reply by lg-nineoneninethreenine** [reply]
Yeah, and please use threads than answers, it could be hard to search for things.

**Reply by yuboabc** [reply]
ok
